content/en/docs/components/training/tftraining.md

```diff
@@ -237,7 +236,7 @@ kubectl -n kubeflow get tfjob mnist -o yaml
 Delete it
```

Comment:
We should change `kubectl -n kubeflow get tfjob mnist -o yaml` to `kubectl -n kubeflow get tfjob tfjob-simple -o yaml`